packages/chip/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@lidofinance/chip",
+  "version": "0.0.0-semantic-release",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "license": "MIT",
+  "homepage": "https://github.com/lidofinance/ui",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lidofinance/ui.git",
+    "directory": "packages/chip"
+  },
+  "bugs": {
+    "url": "https://github.com/lidofinance/ui/issues"
+  },
+  "keywords": [
+    "lido",
+    "lidofinance",
+    "styled-components",
+    "component",
+    "react-component",
+    "react"
+  ],
+  "dependencies": {
+    "@lidofinance/theme": "workspace:*",
+    "@lidofinance/utils": "workspace:*"
+  },
+  "peerDependencies": {
+    "react": "16 || 17",
+    "react-dom": "16 || 17",
+    "react-is": "16 || 17",
```

Comment:
Why do we need it here? Not sure if I see where it's used

Reply:
I see this dependency in all other packages of the library. I will remove it for the chip component.